source/fundamentals/odata.txt

```diff
@@ -0,0 +1,203 @@
+      - Adds the OData service and enables the ``Select()`` query capability.
+      - Registers the route by using the ``AddRouteComponents()`` method.
+      - Calls the ``UseRouting()`` and ``MapControllers()`` methods to match
+        incoming HTTP requests and route them to the appropriate endpoint.
+
+   .. step:: Run the Application
+
+      Run the application by using your IDE, or by running the following command
+      in your shell at the root directory of your project:
+
+      .. code-block:: shell
+
+         dotnet run ODataExample.csproj
+
+      After running the application, your terminal displays output similar
+      to the following:
+
+      .. code-block:: none
+
+         info: Microsoft.Hosting.Lifetime[14]
+               Now listening on: http://localhost:5183
+         info: Microsoft.Hosting.Lifetime[0]
+               Application started. Press Ctrl+C to shut down.
+         info: Microsoft.Hosting.Lifetime[0]
+               Hosting environment: Development
+         info: Microsoft.Hosting.Lifetime[0]
+               Content root path: <Path to your project>
+
+      .. tip::
+
+         After running your application, your IDE might automatically open a
+         browser window to the URL where the application is running, which
+         displays a ``"page can't be found"`` error. This is expected because the
+         application only has a single ``Get`` endpoint configured.
+      
+   .. step:: Query the Data
+
+      To query the data, navigate to the ``Get`` endpoint specified in the
+      application. To do so, open a browser and navigate to the ``localhost``
+      URL specified in the terminal output from the preceding step. Then, append
+      the route for the ``Get`` endpoint: ``/odata/Restaurants``. For example, if an
+      application is running at ``localhost:5183``, navigate to
+      ``http://localhost:5183/odata/Restaurants``.
+
+      If successful, the browser displays 5 restaurants in the
+      collection, in JSON format. The output is similar to the
+      following:
+
+      .. code-block:: json
+
+         {
+           "@odata.context": "http://localhost:5183/odata/$metadata#Restaurants",
+           "value": [
+             {
+               "Name": "Glorious Food",
+               "RestaurantId": "40361521",
+               "Cuisine": "American",
+               "Borough": "Manhattan",
+               "Id": "...",
+               "Address": {
+                 "Building": "522",
+                 "Coordinates": [-73.95171, 40.767461],
+                 "Street": "East   74 Street",
+                 "ZipCode": "10021"
+               },
+               "Grades": [
+                 ...
+               ]
+             },
+
+             ...
+
+             ]
+         }
+
+Additional Information
+----------------------
+
+To learn more about ASP.NET Core OData, see the `Microsoft OData documentation
+<https://learn.microsoft.com/en-us/odata/webapi-8/overview>`__.
```

Comment:
Also we might want to mention that current version does not support OData-Aggregation ($apply query option).

Reply:
Added a note to the section above where query operations are added